R/load_data.R

```diff
@@ -0,0 +1,16 @@
+#'@export
+#'
+
+load_data <- function(){
+  
+  path <- rappdirs::user_data_dir("GeyseRdata.tsv","GeyseRdata")
```

Comment:
I'm not really sure whether this is a good idea 🤔. I'm generally not a huge fan of software putting files in places where I'm not aware of them, and I'm pretty sure most people will not know to look in their data directory after running this function. I'd vote for a simple temporary file with an option to specify a target folder as a parameter. Thoughts?

Reply:
I fixed this and made it so they have the option to specify the location. In the function I also print out where the location is saved to.